test/integration/data/expense_reports/tstdrv2089588.json

```diff
@@ -17,7 +17,7 @@
         "type": "vendor"
     },
     "expenseReportCurrency": {
-        "name": "USD",
+        "name": "USA",
```

Comment:
did this actually work? how can USA be a currency, do we have one in our account?

Reply:
We are using this in netsuite as well as in fixtures